data/wrangler-unenv-polyfills.json

```diff
@@ -10420,7 +10420,7 @@
         "isCryptoKey": "function",
         "isDataView": "function",
         "isDate": "function",
-        "isExternal": "function",
+        "isExternal": "stub",
```

Comment:
is this ok? usually going from function to stub indicates a regression unless previously the function was just a one-off stub that wasn't detectable by our infra.

Reply:
> unless previously the function was just a one-off stub that wasn't detectable by our infra.

I think it's the case here as it's not supported by `workerd`

![Screen Shot 2024-08-29 at 07 21 44 AM](https://github.com/user-attachments/assets/598ad907-4240-4a07-a737-640f485d72eb)